### PR TITLE
fix: remove unselected classes from ICS export

### LINKED ISSUE #51: ICS export includes all lab times
To reproduce:
* Visit https://timetable.cssa.club/?y=2022&s=S1&MATH2320=WorA2&COMP2300=ComA7&COMP4670=ComA4
* Export > any option

All of the lab times show up in your calendar, not just the selected ones

### PR DESCRIPTION
**Pull request by pl4nty**
Fixes #51
Config change allows debugging Functions standalone, or when they crash

### CODE REVIEW

File: api/GetICS/index.js [hunk 1]
```diff
@@ -52,11 +52,14 @@ module.exports = async function (context, req) {
...
+            // Convert client qs format to ANU database lookup
+            // LecA1,LecA2,LecB1 => { LecA: [01, 02], LecB: [01] }
             const selected = req.query[courseCode].split(',').reduce((acc,val) => {
-                const parts = val.split(' ')
-                const activity = acc[parts[0]]
-                acc[parts[0]] = activity === undefined ? [parts[1]] : activity.concat(parts[1])
+                // TutA1 => ['TutA1', 'TutA', '1']
+                const r = val.match(/([^0-9]*)([0-9]+)$/)
+                if (r) {
+                    acc[r[1]] = [...(acc[r[1]] || []), r[2].padStart(2,'0')]
```
Comment (OliverBalfour) [inline api/GetICS/index.js:61]: Could this be `acc[r[1]] = acc[r[1]] || []; acc[r[1]].push(r[2].padStart(2,'0');`? Doing it all in one line is a little trickier to understand imo

**Comment by OliverBalfour** [comment]
Works like a charm!

**Review by OliverBalfour — APPROVED** [review]

**Comment by OliverBalfour** [comment]
Happy to merge these as is, or with the one change I mentioned as long as it's tested on the staging branch

File: api/GetICS/index.js (hunk 1 above)
Comment (pl4nty) [inline api/GetICS/index.js:61]:
```suggestion
                    acc[r[1]] ??= []
                    acc[r[1]].push(r[2].padStart(2,'0'))
```
That's fair, a nullish assignment would be much simpler

**Comment by OliverBalfour** [comment]
LGTM!

**Comment by pl4nty** [comment]
@OliverBalfour oof I hadn't actually tested this locally, it dies - guessing the nullish assignment doesn't work in node 12

**Comment by pl4nty** [comment]
(why SWA Functions are hardstuck on 12, idk)

**Comment by OliverBalfour** [comment]
@pl4nty are you saying this breaks in prod? Should I revert the nullish assignment change as a hotfix?

**Comment by pl4nty** [comment]
> @pl4nty are you saying this breaks in prod? Should I revert the nullish assignment change as a hotfix?
> 

All good, I hotfixed in #70

**Comment by OliverBalfour** [comment]
Perfect, thanks!

**Comment by pl4nty** [comment]
Dw about the build fail on that PR, it just hit the 3 environment cap for staging envs

**Pull request merged**